.gitignore

```diff
@@ -56,3 +57,8 @@ ENV/
 env.bak/
 venv.bak/
 
+# Vim
```

Comment:
Please don't add files specific to your IDE (Use a global ignore in your git config)

Reply:
ok